pkg/common/catalog/fill.go

```diff
@@ -17,6 +17,10 @@ type PluginInfo interface {
 	Name() string
 }
 
+type PluginName string
```

Comment:
Temporary, quick-and-dirty convenience type for implementing `catalog.PluginInfo` with a string representing the plugin name. This type is likely not to be needed  in the refactored catalog system.